kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/config/AbstractProjectConfig.kt

```diff
@@ -297,6 +297,12 @@ abstract class AbstractProjectConfig {
    @ExperimentalKotest
    open var testCoroutineDispatcher: Boolean = Defaults.testCoroutineDispatcher
 
+   /**
+    * If set to false then private spec classes will be ignored by the test engine.
+    * Defaults to true.
+    */
+   open var ignorePrivateClasses: Boolean? = null
```

Comment:
Can it really be null if we are saying the default is true?

Reply:
I also assume the default is false now

Reply:
yep fixed